x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rule_status_filter.tsx

```diff
@@ -59,22 +49,29 @@ export const RuleStatusFilter = (props: RuleStatusFilterProps) => {
     setIsPopoverOpen((prevIsOpen) => !prevIsOpen);
   }, [setIsPopoverOpen]);
 
-  const renderClearAll = () => {
-    return (
-      <div>
-        <EuiButtonEmpty
-          style={{
-            width: '100%',
-          }}
-          size="xs"
-          iconType="crossInACircleFilled"
-          color="danger"
-          onClick={() => onChange([])}
-        >
-          Clear all
-        </EuiButtonEmpty>
-      </div>
-    );
+  const renderRuleStateOptions = (status: 'enabled' | 'disabled' | 'snoozed') => {
+    if (status === 'enabled') {
+      return (
+        <FormattedMessage
+          id="xpack.triggersActionsUI.sections.ruleDetails.ruleStateFilter.enabledOptionText"
+          defaultMessage="Rule is enabled"
+        />
+      );
+    } else if (status === 'disabled') {
+      return (
+        <FormattedMessage
+          id="xpack.triggersActionsUI.sections.ruleDetails.ruleStateFilter.disabledOptionText"
+          defaultMessage="Rule is disabled"
+        />
+      );
+    } else if (status === 'snoozed') {
+      return (
+        <FormattedMessage
+          id="xpack.triggersActionsUI.sections.ruleDetails.ruleStateFilter.snoozedOptionText"
+          defaultMessage="Rule has snoozed"
```

Comment:
Though there seems to be complexity around whether the snooze is active or not, perhaps this is shorter and valid?:
```suggestion
          defaultMessage="Snooze scheduled"
```

Reply:
it is also possible to just have a relative snooze like I only want to snooze this rule for two hours.

Reply:
To me a relative snooze is still something you schedule, so IMO that text would work for both. We can pursue further options in a subsequent PR.